### PR TITLE
Update information about Les-Tilleuls.coop

### PR DESCRIPTION
**Pull request by Taminoful**
I just learned some new information about Les-Tilleuls.coop participation in Hacktoberfest. So I added the sign up link and corrected to blog post to the companies official statement rather than the Owners personal blog as promised in https://github.com/crweiner/hacktoberfest-swag-list/pull/82#issuecomment-537090099.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,8 +88,8 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 
 - **Swag**: An API Platform surprise gift
 - **Requirements**: Create at least 3 PRs to [API Platform repositories](https://github.com/api-platform) during October.
-- **How to sign up**: Not clarified by now.
-- **Notes**: See this announcement from [Kévin Dunglas on his Blog](https://dunglas.fr/2019/09/api-platform-2-5-revamped-admin-new-api-testing-tool-next-js-and-quasar-app-generators-patch-and-json-schema-support-improved-openapi-and-graphql-support/).
+- **How to sign up**: Complete 3 PRs and [sign up here](https://cooptilleuls.typeform.com/to/CG0pLd).
```
Comment (crweiner) [inline README.md]:
```suggestion
- **How to sign up**: Complete 3 PRs and [sign up here](https://cooptilleuls.typeform.com/to/CG0pLd). Pull requests have to be approved by the API Platform Core Team.
```

File: README.md [hunk 2]
```diff
@@ -278,8 +294,8 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
...
```
Comment (crweiner) [inline README.md]:
```suggestion
- **How to sign up**: Complete 3 PRs and [sign up here](https://cooptilleuls.typeform.com/to/CG0pLd). Pull requests have to be approved by the API Platform Core Team.
```

File: README.md [hunk 3]
```diff
@@ -88,8 +88,8 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
...
+- **Notes**: See this announcement from [the companies website](https://les-tilleuls.coop/fr/blog/article/celebrate-hacktoberfest-with-api-platform).
```
Comment (crweiner) [inline README.md]:
```suggestion
- **Notes**: See this announcement from [the company's website](https://les-tilleuls.coop/fr/blog/article/celebrate-hacktoberfest-with-api-platform).
```

File: README.md [hunk 4]
```diff
@@ -278,8 +294,8 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
...
```
Comment (crweiner) [inline README.md]:
```suggestion
- **Notes**: See this announcement from [the company's website](https://les-tilleuls.coop/fr/blog/article/celebrate-hacktoberfest-with-api-platform).
```

**Comment by crweiner** [comment]
Hi @Taminoful! Thanks again for your contribution to the Hacktoberfest Swag List! Please take a look at the comments I made above so we can tweak some small things. Also, can you please rebase your `master` so it pulls in the changes I've made so far (such as move **SalesAgility** to the correct spot in the list).

Thank you!! :smile_cat:

**Comment by Taminoful** [comment]
@crweiner Fixed. 🚀

**Comment by crweiner** [comment]
Wonderful job @Taminoful! Thank you again for another wonderful contribution to the Hacktoberfest Swag List!!!

**Pull request merged**